streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -237,4 +238,9 @@ default boolean commitRequested() {
      */
     Map<TopicPartition, Long> changelogOffsets();
 
+    Map<TopicPartition, Long> getCommittedOffsets();
+
+    Map<TopicPartition, Long> getHighWaterMark();
+
+    Optional<Long> getTimeCurrentIdlingStarted();
```

Comment:
In Kafka, we try to avoid using `get***` pattern to simplify the function signature. Let's remove the `get` part then.